app/controllers/lookups_controller.rb

```diff
@@ -5,13 +5,18 @@ def by_base_path
     states = %w(published unpublished)
     base_paths = params.fetch(:base_paths)
 
-    base_paths_and_content_ids = Edition.with_document
-      .left_outer_joins(:unpublishing)
+    scope = Edition.left_outer_joins(:unpublishing)
+
+    # where not in (..) does not return records that where the field is null
+    scope = scope.where(unpublishings: { type: nil }).or(
+      scope.where.not(unpublishings: { type: params.fetch(:exclude_unpublishing_types, %w{vanish redirect gone}) })
+    )
+
+    scope = scope.with_document
       .where(state: states, base_path: base_paths)
-      .where("state = 'published' OR unpublishings.type = 'withdrawal'")
```

Comment:
From just reading this query, I think this would have had also provided content ids for content that was superseded, provided it was unpublished. I think this is undesirable, I guess because it allows there to be multiple content ids that could be considered valid in the case where there are multiple superseded and unpublished editions with different content ids.

 The changes you have made keep this as a possibility, but I think they also open up the possibility of superseded items which haven't been unpublished being selected.

My suggestion would be to add the condition `.where(content_store: 'live')` to the query, as that will ensure that no superseded content is selected. What do you think?